library.sh

```diff
@@ -517,7 +608,7 @@ function report_go_test() {
   echo "Test log (ANSI) written to ${ansilog}"
 
   htmllog="${logfile/.jsonl/.html}"
-  go_run github.com/buildkite/terminal-to-html/v3/cmd/terminal-to-html@v3.11.0 \
+  go_run github.com/buildkite/terminal-to-html/v3/cmd/terminal-to-html@v3.10.0 \
```

Comment:
Downgrading? I suppose there's some problem with the latest version? Perhaps 3.11.0 requires Golang 1.22...

Reply:
Yes, github.com/buildkite/terminal-to-html/v3/cmd/terminal-to-html@v3.11.0 requires Go 1.22.

I'll bump this back in a follow-up PR.